estela-web/src/components/ProjectJobListPage/index.tsx

```diff
@@ -439,9 +446,14 @@ export class ProjectJobListPage extends Component<RouteComponentProps<RouteParam
             (response: SpiderJobCreate) => {
                 history.push(`/projects/${this.projectId}/spiders/${this.state.spiderId}/jobs/${response.jid}`);
             },
-            (error: unknown) => {
-                console.error(error);
-                incorrectDataNotification();
+            async (error) => {
+                const data = await error.json();
+                if (data.type && data.type === "BillingError") {
```

Comment:
Do not use anything related to billing, you could use a set of conditions that are declared in the external app. If you want to discuss the issue schedule a call.

Reply:
I added that functionality on bitmaker-cloud-microfrontend

Reply:
ok, I added a new function on microfrontend